ui/src-tauri/src/lib.rs

```diff
@@ -78,11 +78,33 @@ fn add_command(command: AddCommand) -> Result<(), String> {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DeleteCommand {
+    pub id: i64,
+}
+
+#[tauri::command]
+fn delete_command(command: DeleteCommand) -> Result<(), String> {
+    let logic = match Logic::try_default() {
+        Ok(l) => l,
+        Err(e) => return Err(format!("Failed to initialize Logic: {:?}", e)),
+    };
+
+    match logic.handle_delete_command(command.id) {
+        Ok(_c) => Ok(()),
+        Err(e) => Err(format!("Error deleting command: {:?}", e)),
+    }
+}
+
```

Comment:
If you want to be more idiomatic, you can do:

```suggestion
fn delete_command(command: DeleteCommand) -> Result<(), String> {
    let logic = Logic::try_default()
        .map_err(|e| format!("Failed to initialize Logic: {:?}", e))?;

    logic
        .handle_delete_command(command.id)
        .map_err(|e| format!("Error deleting command: {:?}", e))
}

```

Reply:
Changed. Also did for the other methods in this file.